src/devices/Units/Pressure.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Units
+{
+    /// <summary>
+    /// Class representing pressure
+    /// </summary>
+    public static class Pressure
```

Comment:
What I meant is Pressure unit similar to Temperature unit with this constant in it.

Reply:
There can be conversions between different units here, eventually. It should be in its own class simply because the mean sea-level pressure has absolutely nothing to do with temperature.

Reply:
I meant to add a full support, fine for now I guess though

Reply:
Complete rewrite made.